docker/Dockerfile.dsv.chainguard

```diff
@@ -0,0 +1,5 @@
+# https://goreleaser.com/customization/docker/?h=scrat#how-it-works
+# trunk-ignore(hadolint/DL3007)
+FROM cgr.dev/chainguard/go:latest
```

Comment:
I guess this base image is too big :thinking:

Reply:
```
REPOSITORY                                                          TAG                  IMAGE ID       CREATED         SIZE   
cgr.dev/chainguard/go                                               latest               3a0426be517b   17 hours ago    536MB 
```
huuuge :smile:

Reply:
Way to catch this.  I used the builder image instead of the base static image by mistake. Thank you! I'll change this and it will be I'm guessing 30mb or less.

Reply:
https://edu.chainguard.dev/chainguard/chainguard-images/reference/static/overview/

This was the one I meant to use.  😆